src/sockperf.cpp

```diff
@@ -3222,7 +3229,7 @@ void do_test() {
     info.fd_num = s_fd_num;
 #ifdef USING_VMA_EXTRA_API
     if (g_vma_api && s_user_params.is_vmazcopyread) {
-        for (int i = s_fd_min; i < s_fd_max; i++) {
+        for (int i = s_fd_min; i <= s_fd_max_zcopy; i++) {
```

Comment:
This code is useless as far as you allocate new object in `prepare_socket()`. So using this cycle you allocate memory for the same key twice.